libs/application/templates/children-residence-change/src/lib/dataSchema.ts

```diff
@@ -36,9 +40,8 @@ export const dataSchema = z.object({
     .string()
     .optional()
     .refine((v) => v && v !== '', error.validation.durationDate.defaultMessage),
-  interview: z.enum(['yes', 'no']).refine((v) => v, {
-    message: error.validation.interview.defaultMessage,
-  }),
+  interview: interview,
+  interviewParentB: interview,
```

Comment:
Could not find out how to use the `InterviewFieldIds` enum here. Any ideas @Rikkiabb ?

Reply:
Does this work: 
`  [InterviewFieldIds.parentA]: interview` ?

Reply:
🤦  Virkar auðvitað 😄